src/controllers/channelController/index.ts

```diff
@@ -0,0 +1,11 @@
+import { getAllChannels } from './getAllChannels';
+import { createChannel } from './createChannel';
+import { addUserToChannel } from './addUserToChannel';
+
+const channelController = {
+  getAllChannels,
+  createChannel,
+  addUserToChannel
+}
+
+export default channelController; 
```

Comment:
refactor channel controller [PART 4] here we barrel all the methods into one object called `channelController` this makes the code more readable later on in the router